modules/bash

```diff
@@ -0,0 +1,40 @@
+# GNU bash
+modules-$(CONFIG_BASH) += bash
```

Comment:
Actually.... If I merge other PRs this is non issue.

```
2023-02-09 12:20:48-05:00 DONE coreboot
# Use coreboot.rom, because custom output files might not be processed by cbfstool
"/home/user/heads/build/x86/coreboot-4.13/t420-hotp-maximized/cbfstool" "/home/user/heads/build/x86/coreboot-4.13/t420-hotp-maximized/coreboot.rom" print
FMAP REGION: COREBOOT
Name                           Offset     Type           Size   Comp
cbfs master header             0x0        cbfs header        32 none
fallback/romstage              0x80       stage           87756 none
cpu_microcode_blob.bin         0x157c0    microcode       26624 none
fallback/ramstage              0x1c040    stage          119325 none
config                         0x392c0    raw               925 none
revision                       0x396c0    raw               697 none
fallback/dsdt.aml              0x399c0    raw             14615 none
vbt.bin                        0x3d340    raw              1368 LZMA (3985 decompressed)
cmos.default                   0x3d900    cmos_default      256 none
cmos_layout.bin                0x3da40    cmos_layout      1992 none
fallback/postcar               0x3e240    stage           27288 none
fallback/payload               0x44d40    simple elf    7678919 none
(empty)                        0x797940   null           164952 none
bootblock                      0x7bfdc0   bootblock       65536 none
2023-02-09 12:20:48-05:00 INSTALL   build/x86/coreboot-4.13/t420-hotp-maximized/coreboot.rom => build/x86/t420-hotp-maximized/heads-t420-hotp-maximized-v0.2.0-1364-g0cbc2eb-dirty.rom
762466a46f1507b423c77ebd5537bb369881b0c37a3c51fcd674dd4a5589dd75  build/x86/t420-hotp-maximized/heads-t420-hotp-maximized-v0.2.0-1364-g0cbc2eb-dirty.rom
762466a46f1507b423c77ebd5537bb369881b0c37a3c51fcd674dd4a5589dd75  /home/user/heads/build/x86/t420-hotp-maximized/heads-t420-hotp-maximized-v0.2.0-1364-g0cbc2eb-dirty.rom
user@heads-tests:~/heads$ git diff
diff --git a/boards/t420-hotp-maximized/t420-hotp-maximized.config b/boards/t420-hotp-maximized/t420-hotp-maximized.config
index 7b0d9008..627c6e3a 100644
--- a/boards/t420-hotp-maximized/t420-hotp-maximized.config
+++ b/boards/t420-hotp-maximized/t420-hotp-maximized.config
@@ -28,6 +28,7 @@ CONFIG_UTIL_LINUX=y
 CONFIG_LVM2=y
 CONFIG_MBEDTLS=y
 CONFIG_PCIUTILS=y
+CONFIG_BASH=y
 
 #Remote attestation support
 #TPM based requirements
diff --git a/boards/x230-maximized/x230-maximized.config b/boards/x230-maximized/x230-maximized.config
index c2f4734f..7a584ed4 100644
--- a/boards/x230-maximized/x230-maximized.config
+++ b/boards/x230-maximized/x230-maximized.config
@@ -26,6 +26,7 @@ CONFIG_UTIL_LINUX=y
 CONFIG_LVM2=y
 CONFIG_MBEDTLS=y
 CONFIG_PCIUTILS=y
+CONFIG_BASH=y
 
 #Remote attestation support
 #TPM based requirements
diff --git a/config/coreboot-t420-hotp-maximized.config b/config/coreboot-t420-hotp-maximized.config
index cb455264..455aa803 100644
--- a/config/coreboot-t420-hotp-maximized.config
+++ b/config/coreboot-t420-hotp-maximized.config
@@ -3,7 +3,7 @@ CONFIG_USE_OPTION_TABLE=y
 CONFIG_STATIC_OPTION_TABLE=y
 CONFIG_VENDOR_LENOVO=y
 CONFIG_NO_POST=y
-CONFIG_CBFS_SIZE=0x750000
+CONFIG_CBFS_SIZE=0x7E7FFF
 CONFIG_IFD_BIN_PATH="@BLOB_DIR@/xx20/ifd.bin"
 CONFIG_ME_BIN_PATH="@BLOB_DIR@/xx20/me.bin"
 CONFIG_GBE_BIN_PATH="@BLOB_DIR@/xx20/gbe.bin"

```

Reply:
With #1298, this is not a problem but for locally built x220/t420 boards (8mb) which don't necessarily include modified ifd + neutered me with freed space relocalized into ifd.

@JonathonHall-Purism : local tests above shows that t420-hotp-maximized is compiling correctly as well.

Reply:
That's awesome to hear @tlaurion :star_struck:   Looking forward to a new era with bash!

Reply:
Will depend on https://github.com/osresearch/heads/pull/1121 being merged. Diffoscope is taking forever to produce intended report prior of review there.